src/utils/user/User.js

```diff
@@ -94,10 +94,15 @@ export class User {
   signIn(
     options = {
       method: AUTH_AUTH0,
-      redirectURL: window.location.href
+      redirectURL: window.location.href,
+      closeAfterSignedIn: false
     },
   ) {
-    const { method, redirectURL } = options;
+    const { method, redirectURL, closeAfterSignedIn } = options;
+    if (closeAfterSignedIn) {
+      accountStorage.setCloseAfterSignedIn();
+    }
+    
```

Comment:
I think removing the signin would be good (in case it is from an earlier time that didnt finish properly)
so something like
   if (closeAfterSignedIn) {
      accountStorage.setCloseAfterSignedIn();
    } else { // new code-
accountStorage.rmCloseAfterSignedIn();
}